stack.yaml

```diff
@@ -1,4 +1,6 @@
 resolver: lts-21.25
+nix:
+  packages: [ pkg-config libz ]
```

Comment:
Is this just for yourself?

Reply:
Just coming by to resolve this conversation -- for posterity, this change allows building on NixOS (as we already discussed).